packages/blockchain-link/src/types/responses.ts

```diff
@@ -47,8 +47,8 @@ export interface Utxo {
     vout: number;
     amount: string;
     blockHeight: number;
-    address: string;
-    path: string;
+    address?: string;
+    path?: string;
```

Comment:
The thing is that you can request utxos for a single address as well, and in that case Blockbook doesn't return neither address (because it's pointless) nor path (because it doesn't know it). The discrepancy in declared and obtained type is e.g. here: https://github.com/trezor/trezor-suite/blob/4024503f1f183fa35a05c8ff3f0236f6732dc859/packages/blockchain-link/src/index.ts#L137.